internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -75,6 +80,7 @@ type BundleReconciler struct {
 //+kubebuilder:rbac:groups=core.rukpak.io,resources=bundles/finalizers,verbs=update
 //+kubebuilder:rbac:groups=core,resources=pods,verbs=list;watch;create;delete
 //+kubebuilder:rbac:groups=core,resources=pods/log,verbs=get
+//+kubebuilder:rbac:groups=core,resources=secrets,verbs=get
```

Comment:
This can also be moved to the git+auth follow-up PR, right?

Reply:
As an aside, we might want to maintain this rule separately because we'll only need secret access in the rukpak-system namespace, so we just need a role/rolebinding.